content/tutorials/MQTT-snake-js.textile

```diff
@@ -0,0 +1,268 @@
+---
+title: "MQTT Tutorial: Snake with Javascript"
+section: tutorials
+index: 104
+languages:
+  - javascript
+  - nodejs
+---
+
+Ably Realtime provides support for a number of protocols with its pub/sub system, one of which is MQTT. MQTT is an open ISO standard providing a lightweight messaging protocol for small sensors and mobile devices, optimized for high-latency or unreliable networks. Whilst we recommend use of the Ably realtime protocol where you can, MQTT can be great for use with languages we don't currently support, as well as when you stringent bandwidth restrictions.
```

Comment:
Can you please link MQTT to a suitable resource at least once?
The last sentence is missing a "have" before stringent.  
I also think the last sentence is not quite clear enough.  We are telling people to use the Ably realtime protocol, yet not explaining why.  The reasons include that we can provide a far richer feature set such as presence, automatic encoding and decoding of data types, etc.  Please also explain why then someone would choose MQTT over Ably.  Finally, users aren't really using the Ably realtime protocol, but are instead using our "Ably client library SDKs" which should be a link to our download page